include/tinycrypt/ecc.h

```diff
@@ -68,6 +68,7 @@
  *
  */
 
+#if defined(MBEDTLS_USE_UECC)
```

Comment:
Is it possible to exclude `tinycrypt` from the CMake library list when `MBEDTLS_USE_UECC` is unset? So far, we're always building _some_ `tinycrypt/libtinycrypt.a`, but it'll be empty if `MBEDTLS_USE_UECC` is unset.

Reply:
I am not sure if cmake or make depending on config.h is a good idea. How complex mechanism we want for enabling this feature? This isn't going to be backported on the mainstream mbedtls anyway. @mpg what is your opinion on this?

Reply:
I also don't think having `cmake` depend on `config.h` is a good idea, or even doable at all: what if the user invokes `cmake` first, then adjusts `config.h`? Also, the problem (as far as it is a problem) is not really new: if you disable `MBEDTLS_SSL_TLS_C` (and `NET_C` and `DEBUG_C`) then libmbedtls.a is going to be pretty empty too. Also, as Jarno says, this is just for this branch, and as discussed in the chat, we might not want to spend too much time on tuning build systems, so I'm quite inclined to leave it this way.

(Even if my personal preference would be to have cmake do the same as make here, ie include those files in libmbedcrypto.a and never build libtinycrypt.a, but that's not something I requesting because I think there are better uses of our time than polishing the build system here.)

Reply:
@mpg @jarlamsa Alright, thanks to you both for commenting! I'm fine to leave things as they are, then.